karma.conf.js

```diff
@@ -23,11 +23,11 @@ module.exports = function(config) {
             platform: 'iOS',
             sdk: '10.0'
         },
-        Safari_IOS_12: {
+        Safari_IOS_11: {
             base: 'MobileSafari',
             name: 'iPhone 5s',
             platform: 'iOS',
-            sdk: '12.1'
+            sdk: '11.4'
```

Comment:
@niklasvh This is a regression of sorts; but until it's [confirmed](https://github.com/actions/virtual-environments/issues/717) whether updated `macOS` vmImages contain support for `iPhone 5s` on `iOS 12`, this seems to fix the build while maintaining support for iPhone 5s devices.